diff --git a/dist/frappe-gantt.es.js b/dist/frappe-gantt.es.js
index e7b7b7d3ec71db0086c4d989a3eb6d9656ab78dd..21208f16163bf4aa568a2a7cbbccccbcd2ccecca 100644
--- a/dist/frappe-gantt.es.js
+++ b/dist/frappe-gantt.es.js
@@ -1364,7 +1364,7 @@ class B {
     ) / this.config.step * this.config.column_width;
     this.$container.scrollTo({
       left: i - this.config.column_width / 6,
-      behavior: "smooth"
+      behavior: "instant"
     }), this.$current && this.$current.classList.remove("current-upper"), this.current_date = d.add(
       this.gantt_start,
       this.$container.scrollLeft / this.config.column_width,
@@ -1468,12 +1468,12 @@ class B {
     };
     let g = 0;
     if (p.on(this.$svg, "mousemove", ".bar-wrapper, .handle", (l) => {
-      this.bar_being_dragged === !1 && Math.abs((l.offsetX || l.layerX) - g) > 10 && (this.bar_being_dragged = !0);
+      this.bar_being_dragged === !1 && Math.abs((l.clientX - this.$svg.getBoundingClientRect().left) - g) > 10 && (this.bar_being_dragged = !0);
     }), p.on(this.$svg, "mousedown", ".grid-column", (l) => {
       this.trigger_event("date_click", [this.getDateFromClick(l)]);
     }), p.on(this.$svg, "mousedown", ".bar-wrapper, .handle", (l, _) => {
       const c = p.closest(".bar-wrapper", _);
-      _.classList.contains("left") ? (s = !0, _.classList.add("visible")) : _.classList.contains("right") ? (r = !0, _.classList.add("visible")) : _.classList.contains("bar-wrapper") && (t = !0), this.popup && this.popup.hide(), e = l.offsetX || l.layerX, a = c.getAttribute("data-id");
+      _.classList.contains("left") ? (s = !0, _.classList.add("visible")) : _.classList.contains("right") ? (r = !0, _.classList.add("visible")) : _.classList.contains("bar-wrapper") && (t = !0), this.popup && this.popup.hide(), e = l.clientX - this.$svg.getBoundingClientRect().left, a = c.getAttribute("data-id");
       let u;
       this.options.move_dependencies ? u = [
         a,
@@ -1552,7 +1552,7 @@ class B {
       }));
     }), p.on(this.$svg, "mousemove", (l) => {
       if (!h()) return;
-      const _ = (l.offsetX || l.layerX) - e;
+      const _ = (l.clientX - this.$svg.getBoundingClientRect().left) - e;
       o.forEach((c) => {
         const u = c.$bar;
         u.finaldx = this.get_snap_position(_, u.ox), this.hide_popup(), s ? a === c.task.id ? c.update_bar_position({
@@ -1576,7 +1576,7 @@ class B {
   bind_bar_progress() {
     let t = 0, e = null, i = null, s = null, r = null;
     p.on(this.$svg, "mousedown", ".handle.progress", (o, h) => {
-      e = !0, t = o.offsetX || o.layerX;
+      e = !0, t = o.clientX - this.$svg.getBoundingClientRect().left;
       const l = p.closest(".bar-wrapper", h).getAttribute("data-id");
       i = this.get_bar(l), s = i.$bar_progress, r = i.$bar, s.finaldx = 0, s.owidth = s.getWidth(), s.min_dx = -s.owidth, s.max_dx = r.getWidth() - s.getWidth();
     });
@@ -1586,7 +1586,7 @@ class B {
     ]);
     p.on(this.$svg, "mousemove", (o) => {
       if (!e) return;
-      let h = o.offsetX || o.layerX;
+      let h = o.clientX - this.$svg.getBoundingClientRect().left;
       if (h > t) {
         let _ = a.find(
           ([c, u]) => h >= c && h < u
